readme.md

```diff
@@ -2,29 +2,92 @@
 
 [![Build](https://github.com/Letterbook/Letterbook/actions/workflows/pull-request.yml/badge.svg?branch=main)](https://github.com/Letterbook/Letterbook/actions/workflows/pull-request.yml)
 
-Letterbook is intended to be a mastodon-compatible federated microblogging server. The project has three primary goals:
+Letterbook is a federated microblogging service, implementing ActivityPub. The project was started with the hope to make hosting a fediverse server a better and more sustainable experience. We also want to make it a safer and more inclusive space for the people who make the fediverse their social media home. Those are big aspirations, but we have some solid plans about how to start, and [we would love more input about where to go from there](#contributing).
 
-1. Functional compatibility with Mastodon, and Mastodon clients
-2. Reduced cost and complexity to deploy and operate
-3. More robust moderation tools
+## Features
+I hope it goes without saying, but we will support most of the features you use regularly on other fedi apps. Things like hashtags, custom emojis, lists, filters, editable posts, account migration, and so on.
```

Comment:
This might be a little too broad. There's a *lot* of features out there on fedi, and different people have different opinions about which ones are common. I'm worried that it could lead to demands to implement all kinds of obscure features.

Reply:
That's fair